README.md

```diff
@@ -87,7 +87,7 @@ False
 ```
 
 Now go read the
-[Dhall tutorial](https://hackage.haskell.org/package/dhall-1.0.1/docs/Dhall-Tutorial.html)
+[Dhall tutorial](https://hackage.haskell.org/package/dhall-1.1.0/docs/Dhall-Tutorial.html)
```

Comment:
You can make this future proof by changing it to https://hackage.haskell.org/package/dhall/docs/Dhall-Tutorial.html